datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/DatasetComparator.scala

```diff
@@ -165,10 +165,9 @@ class DatasetComparator(dataFrameReference: DataFrame,
   def checkSchemas(testedDF: ComparisonPair[DataFrame], schema: StructType): Unit = {
     val expectedSchema: StructType = getSchemaWithoutMetadata(testedDF.reference.schema)
     val actualSchema: StructType = getSchemaWithoutMetadata(testedDF.actual.schema)
-
-    if (!(SchemaUtils.isSubset(schema, actualSchema) && SchemaUtils.isSubset(schema, expectedSchema))) {
-      val diffSchema = SchemaUtils.diffSchema(schema, actualSchema) ++
-        SchemaUtils.diffSchema(schema, expectedSchema)
+    if (!schema.isSubset(actualSchema) && !schema.isSubset(expectedSchema)) {
```

Comment:
Why not to use `isEquivalent` as above?

Reply:
This is checking if the `schema` is a subset of 2 different schemas; `actualSchema` and `expectedSchema`. No equivalency check going on here.

Reply:
Ah, see. Sorry